content/en/docs/reference/commands/istioctl/index.html

```diff
@@ -3191,106 +3191,6 @@ <h3 id="istioctl-experimental-revision-tag-set Examples">Examples</h3>
  kubectl rollout restart deployments -n test-ns
 
 </code></pre>
-<h2 id="istioctl-experimental-uninstall">istioctl experimental uninstall</h2>
```

Comment:
I think this is due to the rebase issue, if you sync to the lastest master branch, this can be resolved. I recently deleted the `istioctl experimental uninstalll` in the istio repo.

Reply:
FYI https://github.com/istio/istio.io/pull/13487

Reply:
FYI https://github.com/istio/istio.io/pull/13487